### PR TITLE
Add p5play

### PR DESCRIPTION
**Pull request by mvicky2592**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I am not the sole author or employee of a company who created either the topic I am modifying/adding or the collection entry I am modifying/adding.

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [x] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [ ] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

> Please replace this line with an explanation of why you think this topic or collection should be curated.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [x] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [x] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs

p5play is a JavaScript game engine with +50,000 monthly users. https://p5play.org

Notably it's been used at Girls Who Code summer camps and CodeHS courses. v1 is also used by Code.org.

### CODE REVIEW

File: collections/javascript-game-engines/index.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ items:
  - 4ian/GDevelop
  - mrdoob/three.js
  - phoboslab/Impact
+ - quinton-ashley/p5play
```
Comment (ahpook) [inline collections/javascript-game-engines/index.md:22]: New entries need to be appended to the bottom of the list - please put this down below and we can merge it.

**Review by ahpook — CHANGES_REQUESTED** [review]
New entry needs to be at the bottom of the list

**Review by ahpook — APPROVED** [review]
Thanks!

File: collections/javascript-game-engines/index.md (hunk 1 above)
Comment (Natthakchai-Krit) [reply collections/javascript-game-engines/index.md]:
```

```

**Review by Natthakchai-Krit — COMMENTED** [review]
GG

**Pull request merged**